examples/routing/mini_gamma_IRF.jl

```diff
@@ -0,0 +1,52 @@
+using ClimaRivers
+using JSON, Dates
+
+# build hillslope model
+shape = 1.5
+timescale = 1.0
+t_max_hs = 60.0
+hillslope = MizurouteHillslopeV1{Float64}(shape, timescale, t_max_hs)
+
+# build channel model
+wave_velocity = 1.5 * 86400
+diffusivity = 8000 * 86400
+t_max_ch = 120.0
+channel = MizurouteChannelV1{Float64}(wave_velocity, diffusivity, t_max_ch)
+
+# build model
+river_model = HillslopeChannelRiverModel(hillslope, channel)
+
+# build static environment
+routing_lv_basins_dir = "/groups/esm/achiang/ClimaRivers.jl/mini_data/routing/routing_lvs/routing_lvs_lv05"
+attributes_dir = "/groups/esm/achiang/ClimaRivers.jl/mini_data/routing/attributes/attributes_lv05"
+static_env = StaticEnvironment(routing_lv_basins_dir, attributes_dir)
+
+# build dynamic environment
+timeseries_dir = "/groups/esm/achiang/ClimaRivers.jl/mini_data/routing/timeseries/timeseries_lv05"
+output_dir = "/groups/esm/achiang/ClimaRivers.jl/mini_data/routing/simulations/simulations_lv05/gamma_IRF"
+dynamic_env = DynamicEnvironment(timeseries_dir, output_dir)
+
+# build environment
+graph_dict = JSON.parsefile(
+    "/groups/esm/achiang/ClimaRivers.jl/mini_data/routing/graphs/graph_lv05.json",
+)
+env = Environment(static_env, dynamic_env, graph_dict)
+
+## evolutionary model, evolving a state over time
+model_types = ["instant"]
+model_type = model_types[1]
+
+start_date = Date("1996-01-01", "yyyy-mm-dd")
+end_date = Date("2014-12-31", "yyyy-mm-dd")
+dates = collect(start_date:Day(1):end_date)
+
+# streamflow = zeros(dates,basins)
+hillslope_data = zeros(10, 10)  # Replace with actual data once implemented
```

Comment:
Im not sure I understand this "# Replace with actual data once implemented" or understand where the (10,10) sizing comes from. 10 basins, 10 times? should one of these be length(dates)?

Reply:
discuss in meeting (12/6)